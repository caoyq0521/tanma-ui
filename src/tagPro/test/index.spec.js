import { mount } from '@vue/test-utils';
import TmTagPro from '../tagPro.vue';

describe('tmTagPro', () => {
  it('render tagPro', () => {
    const wrapper = mount(TmTagPro);
    expect(wrapper).toMatchSnapshot();
  });
})
  