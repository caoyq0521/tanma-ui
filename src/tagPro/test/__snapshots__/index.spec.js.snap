// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tmTagPro render tagPro 1`] = `
<span class="tm-tag tm-tag-pro tm-tag-pro--employee"><!----> <span class="tm-tag-pro__icon"><i class="tm-icon-touxiang "></i></span>
员工标签
<!----></span>
`;
